src/pages/DefaultOrgSelector/DefaultOrgSelector.jsx

```diff
@@ -47,7 +47,7 @@ const renderItem = ({ item }) => {
   if (item?.isProvider) {
     return (
       <div className="flex h-8 items-center gap-2">
-        <A pathname={{ pageName: 'codecovAppInstallation' }}>
+        <A to={{ pageName: 'codecovAppInstallation' }}>
```

Comment:
interestingly, this was the only spot I could find that was using something outside of "href" or "to." Just changed it to "to" for parity